suite-common/wallet-core/src/fiat-rates/fiatRatesActions.ts

```diff
@@ -0,0 +1,26 @@
+import { createAction } from '@reduxjs/toolkit';
+
+import { FiatCurrencyCode } from '@suite-common/suite-config';
+import { TickerId } from '@suite-common/wallet-types';
+
+export const FIAT_RATES_MODULE_PREFIX = '@common/wallet-core/fiat-rates';
+
+type AddFiatRatesForTimestampsPayload = {
+    ticker: TickerId;
+    localCurrency: FiatCurrencyCode;
+    rates: Array<{
+        rate?: number;
+        timestamp: number;
+    }>;
+};
+
+const addFiatRatesForTimestamps = createAction(
```

Comment:
Why not use fullfiled action from thunk?

Reply:
761fa4dd09729d6908b35e0a339a4279e1eb3516